test/scripts/plot_data.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
plot_grf_data  and plot_grf_data_config?

Reply:
Done